steps/bitbucket-server-post-build-status/0.0.1/step.yml

```diff
@@ -0,0 +1,68 @@
+title: Bitbucket server post build status
+summary: |
+  Post build status to bitbucket server
+description: |
+  Post build status to bitbucket server
+website: https://github.com/teameh/bitrise-step-bitbucket-server-post-build-status
+source_code_url: https://github.com/teameh/bitrise-step-bitbucket-server-post-build-status
+support_url: https://github.com/teameh/bitrise-step-bitbucket-server-post-build-status/issues
+published_at: 2019-12-09T18:32:12.393477+01:00
+source:
+  git: https://github.com/teameh/bitrise-step-bitbucket-server-post-build-status.git
+  commit: 1bae4e1e3014c9a4a18a522a9de2a35b3e8e105d
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: false
+is_always_run: true
+is_skippable: false
+inputs:
+- bitbucket_server_domain: $BITBUCKET_SERVER_DOMAIN
+  opts:
+    is_required: true
+    summary: Full domain name without protocol eg 'my-domain.com'
+    title: Bitbucket Server domain name
+- bitbucket_server_username: $BITBUCKET_SERVER_USERNAME
```

Comment:
variable is not exposed by Bitrise CI nor by any step in steplib

Reply:
Fixed